store/generated_key/memstore/cli.go

```diff
@@ -67,18 +76,25 @@ func CLIFlags(envPrefix, category string) []cli.Flag {
 		},
 		&cli.DurationFlag{
 			Name:     PutLatencyFlagName,
-			Usage:    "Artificial latency added for memstore backend to mimic EigenDA's dispersal latency.",
+			Usage:    "(FOR TESTING) Artificial latency added for memstore backend to mimic EigenDA's dispersal latency.",
```

Comment:
nit - isn't this entire construction used for testing? maybe less redundant to put disclosure in category description instead

Reply:
[55b4ec1](https://github.com/Layr-Labs/eigenda-proxy/pull/289/commits/55b4ec1364979030da269e0ebd9f086e1b88e8b7)